views/index.html

```diff
@@ -37,25 +44,59 @@
               'http://xmlns.com/foaf/0.1/img',
             ],
             queryMethod: GraphExplorer.SparqlQueryMethod.GET
-          }, GraphExplorer.OWLStatsSettings),
+          }, SparqlDialect),
         });
 
         /**
         * get the '?resource' search param and load that resource.
         */
-        let url = new URL(window.location.href);
-        let resource = url.searchParams.get("resource");
+        const url = new URL(window.location.href);
+        const resource = url.searchParams.get("resource");
         if (resource != null) {
-          let elm = workspace.getModel().dataProvider.elementInfo({elementIds: [resource]});
+          const elm = workspace.getModel().dataProvider.elementInfo({ elementIds: [resource] });
             elm.then(function(arg) {
             workspace.getModel().createElement(arg[resource])
             workspace.forceLayout();
           });
         }
```

Comment:
This is not useful, since we can already take care of it by using `?resources`.
This should get removed.

Reply:
where do we take care of this?

Reply:
@semanticfire That goal here is to have the possibility to provide either "one" resource or a list of resources? What happens if we provide multiple times the parameter 'resource'? 

That is the behaviour you use in your other deployments?

Reply:
You can either provide a single resource or multiple one by passing them to the `resources` parameter.

Or we can use multiple time the `resource` parameter and get them like this:

```js
(new URL('http://example.com?resource=a&resource=b&resource=c')).searchParams.getAll('resource')
```

this give the following result: `['a', 'b', 'c']`.

@semanticfire and @l00mi: please make a choice between `resources` or providing multiple time `resource`; having both is not a good idea, since this will confuse users.

Reply:
Lets stick to resources where 1 item works as well